### ISSUE TITLE
most_recent cannot be combined with excluded_fields

### ISSUE THREAD

**Issue by dgilge** [treyhunner/django-simple-history#333]
https://github.com/treyhunner/django-simple-history/blob/35febc338fb42110afb6dc15db2d03c8a00ac43e/simple_history/manager.py#L53

Do I see it correctly, that you get a `FieldError` when using `most_recent` in combination with `excluded_fields`?

For `self.get_querset()` returns a query set of historical objects but `self.instance` is the current object. Therefore `self.instance._meta.fields` (line 46) will have more fields than the hisorical objects and you will get:

```
django.core.exceptions.FieldError: Cannot resolve keyword 'some_excluded_field' into field. Choices are: ...
```

See also my [test repository](https://github.com/dgilge/test_simple_history).

**Comment by rossmechanic** [comment]
@dgilge good catch! That should be iterating through only the fields in that are included. Feel free to put up a fix for this bug, and I'd be happy to merge.

**Comment by rossmechanic** [comment]
Fixed by #477. Thanks @Alig1493 !
